src/Highlight.ts

```diff
@@ -54,7 +63,7 @@ export default class Highlight {
   }
 
   public removeStyle() {
-    const {style} = this.data;
+    const { style } = this.data;
```

Comment:
please avoid changing unrelated whitespace, it makes the diff harder to read, if it makes your life easier you can try using the `git add -p` option that allows you to stage chunks of diffs instead of whole files